### ISSUE TITLE
How can I configure my own recipe repo when using lineman fetch <recipe name>

### ISSUE THREAD

**Issue by dematic-rodrigo-silveira** [linemanjs/lineman#374]
I noticed that *fetcher* supports the configuring the recipe repo. The same is not true with *lineman fetch <recipe name>*, as seen in the code below:
```coffee
grunt = require('./../requires-grunt').require()
fetcher = require('fetcher')

module.exports = (recipes = [], cb) ->
  return displayHelp() if recipes.length == 0
  fetcher recipes, {recipeRepo: "https://github.com/linemanjs/fetcher-recipes.git"}, (er) ->
    grunt.warn(er) if er?
    return cb?(er) if er?
    cb?(null)

displayHelp = ->
  grunt.warn """

            Usage: `lineman fetch <recipe_name>`

            For a list of available recipes, see:
            https://github.com/linemanjs/fetcher-recipes/tree/master/recipes

            """
```

I would like to manage the recipes.

Thanks in advance for your help.

**Comment by davemo** [comment]
Hi @dematic-rodrigo-silveira, I added a PR that supports this from the command-line; I'm not sure it's ideal as it may be nicer to define the fetcher recipe url override in the lineman config, or maybe even just a .dotfile in the repo root, but it does work. I'd like some input from @searls and @jasonkarns on whether they have suggestions on how to make this work.

**Comment by davemo** [comment]
Hi @dematic-rodrigo-silveira, I've released `0.36.4` which allows you to override the fetcher recipes repo. Details here: https://github.com/linemanjs/lineman/releases/tag/0.36.4

Hope this works out for you :)
